source/classes.tex

```diff
@@ -3305,16 +3305,17 @@
 \pnum
 \indextext{\idxcode{union}!global anonymous}%
 \indextext{scope!anonymous \tcode{union} at namespace}%
-Anonymous unions declared in a named namespace or in the global
-namespace shall be declared \tcode{static}. Anonymous unions declared at
-block scope shall be declared with any storage class allowed for a
-block-scope variable, or with no storage class. A storage class is not
-allowed in a declaration of an anonymous union in a class scope.
+Anonymous unions declared in a named namespace or in the global namespace
+shall be declared \tcode{static}.
+A block-scope anonymous union declaration shall not
+contain a \grammarterm{storage-class-specifier} that is not
```

Comment:
contain -> have

Otherwise I keep wondering whether you mean "recursively contain", and whether that's even possible. Not desirable here.